product/toolbars/container_service_center_tb.yaml

```diff
@@ -29,6 +29,12 @@
     :title: Monitoring
     :text: Monitoring
     :items:
+    - :button: container_service_timeline
+      :image: timeline
```

Comment:
@dkorn shouldn't this be:

    :icon: product product-timeline fa-lg

cc @zeari